examples/hotrod/pkg/tracing/mux.go

```diff
@@ -41,36 +40,16 @@ func NewServeMux(copyBaggage bool, tracer opentracing.Tracer, logger log.Factory
 type TracedServeMux struct {
 	mux         *http.ServeMux
 	copyBaggage bool
-	tracer      opentracing.Tracer
+	tracer      trace.TracerProvider
 	logger      log.Factory
 }
 
 // Handle implements http.ServeMux#Handle, which is used to register new handler.
 func (tm *TracedServeMux) Handle(pattern string, handler http.Handler) {
 	tm.logger.Bg().Debug("registering traced handler", zap.String("endpoint", pattern))
 
-	middleware := nethttp.Middleware(
-		tm.tracer,
-		handler,
-		nethttp.OperationNameFunc(func(r *http.Request) string {
-			return "HTTP " + r.Method + " " + pattern
-		}),
-		// Jaeger SDK was able to accept `jaeger-baggage` header even for requests without am active trace.
-		// OTEL Bridge does not support that, so we use Baggage propagator to manually extract the baggage
-		// into Context (in otelBaggageExtractor handler below), and once the Bridge creates a Span,
-		// we use this SpanObserver to copy OTEL baggage from Context into the Span.
-		nethttp.MWSpanObserver(func(span opentracing.Span, r *http.Request) {
-			if !tm.copyBaggage {
```

Comment:
@yurishkuro I did workaround [here ](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/instrumentation/net/http/otelhttp/v0.42.0/instrumentation/net/http/otelhttp/handler.go#L64). I think otel sdk doesn't support for `jaeger-baggage` header. Anyway to handle such? 🤔

Reply:
Did you read about this part in my blog post? This isn't about jaeger-baggage header, but about the bridge not allowing baggage-only, without a valid span (which doesn't come from the front end).

Reply:
I see. I read that a few days ago. TBH, but made my assumptions on it. thanks
![image](https://github.com/jaegertracing/jaeger/assets/11625672/39388cf4-5978-4e07-92f7-0c56de3e772a)

Reply:
underneath, I found that. Thanks